test/testdata/namer/redefines_module_as_static_field.rb.symbol-table-raw.exp

```diff
@@ -4,8 +4,8 @@ class <C <U <root>>> < <C <U Object>> ()
       argument <blk><block> @ Loc {file=test/testdata/namer/redefines_module_as_static_field.rb start=??? end=???}
   module <C <U T>> < <C <U Sorbet>>::<C <U Private>>::<C <U Static>>::<C <U ImplicitModuleSuperclass>> () @ (Loc {file=test/testdata/namer/redefines_module_as_static_field.rb start=3:1 end=3:9}, Loc {file=https://github.com/sorbet/sorbet/tree/master/rbi/sorbet/t.rbi start=removed end=removed})
     module <C <U T>>::<C <U AbstractUtils>> < <C <U Sorbet>>::<C <U Private>>::<C <U Static>>::<C <U ImplicitModuleSuperclass>> () @ (Loc {file=test/testdata/namer/redefines_module_as_static_field.rb start=22:1 end=22:24}, Loc {file=https://github.com/sorbet/sorbet/tree/master/rbi/sorbet/t.rbi start=removed end=removed})
-      module <C <U T>>::<C <U AbstractUtils>>::<C <U Methods>> < <C <U Sorbet>>::<C <U Private>>::<C <U Static>>::<C <U ImplicitModuleSuperclass>> () @ Loc {file=test/testdata/namer/redefines_module_as_static_field.rb start=30:3 end=30:28}
-        module <C <U T>>::<C <U AbstractUtils>>::<C <U Methods>>::<C <U CallValidation>> < <C <U Sorbet>>::<C <U Private>>::<C <U Static>>::<C <U ImplicitModuleSuperclass>> () @ Loc {file=test/testdata/namer/redefines_module_as_static_field.rb start=34:3 end=34:44}
+      module <C <U T>>::<C <U AbstractUtils>>::<C <U Methods>> < <C <U Sorbet>>::<C <U Private>>::<C <U Static>>::<C <U ImplicitModuleSuperclass>> () @ Loc {file=test/testdata/namer/redefines_module_as_static_field.rb start=38:3 end=38:28}
+        module <C <U T>>::<C <U AbstractUtils>>::<C <U Methods>>::<C <U CallValidation>> < <C <U Sorbet>>::<C <U Private>>::<C <U Static>>::<C <U ImplicitModuleSuperclass>> () @ Loc {file=test/testdata/namer/redefines_module_as_static_field.rb start=38:3 end=38:44}
```

Comment:
This changes the subsequent loc for an erroneous declaration. If you look at the test -- it's quite an edge case, so I think this is a safe change.